### DISCUSSION TITLE
[General] Top menu doesn't work on zksync.io

### DISCUSSION THREAD

**Discussion by gagarin55** [zkSync-Community-Hub/zksync-developers#123] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.
![zksync_bug](https://github.com/zkSync-Community-Hub/zkync-developers/assets/18405789/e60720c8-4e85-442f-b620-b5db53c9d7b0)

### What is the specific issue or error you're encountering?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

### Can you share the error messages or logs you're receiving, if any?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

### Have you made any recent changes to the contract before encountering this issue?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

### Are there any external libraries or contracts that your contract interacts with?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

### Can you provide the relevant portions of your contract code where the issue is occurring?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

### Have you tried to isolate the problem, and if so, what were the results?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

### What steps have you already taken to try to resolve the issue?

I hope you guys feel well. Your TOP menu on zksync.io doesn't work in Chrome Version 116.0.5845.140 (Official Build) (64-bit) on Linux machine. Wish you to find good JS-dev.

**Comment by JackHamer09** [comment, marked as answer]
Hey @gagarin55, sorry to hear that the navigation menu isn't working for you.
As a solution, please try clearing your browser's cache. Also, make sure you don't have any extensions installed that might affect website behavior.
Let me know if you're still facing this problem.
